ember-cli-build.js

```diff
@@ -300,7 +300,15 @@ module.exports = function(options) {
   });
 
   emberTestingBundle = concat(emberTestingBundle, {
-    outputFile: 'ember-testing.js'
+    outputFile: 'ember-testing.js',
+    hasBootstrap: false,
+    footer: stripIndent`
+      var testing = requireModule('ember-testing');
+      Ember.Test = testing.Test;
```

Comment:
Yes, due to [these lines](https://gist.github.com/trentmwillis/95afa3bcfda5f8d0657435346cd36444#file-ember-testing-js-L19-L23). I originally had `var Ember = requireModule('ember-metal/core');` but that was flagged as a redeclaration.